packages/hoprd/releases.json

```diff
@@ -1,7 +1,7 @@
 {
-  "master": {
+  "rotsee": {
```

Comment:
Actually `rotsee` is not a release name. Should it be `staging` or `development`?

Reply:
Ok, good catch. Thank you